### DISCUSSION TITLE
Melee_Damage_Multiplier and Melee_Repair_Multiplier

### DISCUSSION THREAD

**Discussion by Ongjming** [SmartlyDressedGames/Unturned-3.x-Community#3285] [Q&A] [answered]
I saw this in a news post from a bit back (3.22.9.0 Update Notes). Was wondering where are these applicable/where are they used in?

Melee weapon with repairing function (Like the vanilla blowtorch) have a "Repair" line in its .dat file.
I looked at a few vehicle, body armour and other melee weapons, can't really find the lines on first glance through.
So was hoping to satisfy my curiosity here, plus anyone who might have the same question can see this in the future.

Thanks!

**Comment by SDGNelson** [comment, marked as answer]
It was added to the gameplay config so that the repair efficiency could be changed globally
